app/src/main/java/com/randomappsinc/studentpicker/home/HomeActivity.java

```diff
@@ -208,20 +210,27 @@ protected void onResume() {
     public void onActivityResult(int requestCode, int resultCode, Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
         if (requestCode == IMPORT_FILE_REQUEST_CODE) {
-            if (resultCode == Activity.RESULT_OK && data != null && data.getData() != null) {
-                Uri uri = data.getData();
-
-                // Persist ability to read from this file
-                int takeFlags = data.getFlags()
-                        & (Intent.FLAG_GRANT_READ_URI_PERMISSION
-                        | Intent.FLAG_GRANT_WRITE_URI_PERMISSION);
-                getContentResolver().takePersistableUriPermission(uri, takeFlags);
-
-                String uriString = uri.toString();
-                Intent intent = new Intent(this, ImportFromTextFileActivity.class);
-                intent.putExtra(Constants.FILE_URI_KEY, uriString);
-                startActivity(intent);
-            }
+            maybeOpenImportFileActivity(resultCode,data,true);
```

Comment:
Small spacing thing.
```suggestion
            maybeOpenImportFileActivity(resultCode, data, true);
```